### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by amscad**
Add the ability to filter by topic, or filter by author - as a comma separated string

### CODE REVIEW

File: mytelegraph-feed.yaml [hunk 1]
```diff
@@ -135,6 +136,22 @@ paths:
           required: false
           default: false
           enum: ["true", "false"]
+        - name: filterByTopic
```
Comment (adrianvit) [inline mytelegraph-feed.yaml]: We just need to make sure these are properly url encoded/decoded, as they contain special characters.

**Comment by CristianDina** [comment]
Is there a JIRA task on mytelegraph-feed for implementing filters on feed?

**Comment by amscad** [comment]
The JIRA ticket is - PLAT-1876.  I have renamed the ticket to My Feed - Update endpoint to provide ability to filter the articles a user follows by a topic(s)

**Comment by CristianDina** [comment]
Please don't merge this pull request until we finish with the deploy to prod of the filter inactive features as this will make functionality tests to fail in the jenkins pipeline. After that I can look over the task on feed api.

**Comment by adrianvit** [comment]
@amscad It's ok to merge this now.

**Pull request merged**